holoviews/plotting/bokeh/element.py

```diff
@@ -421,7 +421,7 @@ def _axis_props(self, plots, subplots, element, ranges, pos, *, dim=None,
             specs = ((dim.name, dim.label, dim.unit),)
         else:
             if isinstance(self, OverlayPlot):
-                l, b, r, t = self.get_extents(range_el, ranges, dim)
+                l, b, r, t = self.get_extents(range_el, ranges, range_type=dim or "combined")
```

Comment:
I think that was simply incorrect and should have been:

```suggestion
                l, b, r, t = self.get_extents(range_el, ranges, dimension=dim)
```